frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -122,6 +122,7 @@ interface IManageHostsProps {
 }
 
 const CSV_HOSTS_TITLE = "Hosts";
+const PAGE_SIZE = 50;
```

Comment:
Can we use (and update the number from 20 to 50) `DEFAULT_PAGE_SIZE` on Line 96 on HostTableConfig.tsx instead of a new constant?